src/server/controllers/users.controller.js

```diff
@@ -107,11 +107,12 @@ export function localSignUp(req, res, next) {
       if (existingUser) {
         res.status(201).json({ msg: userCallback.ERR_EXISTING_EMAIL });
       } else {
-        registrationData.deviceToken = req.body.deviceToken;
         return User(registrationData).save();
       }
     })
     .then(registeredUser => {
+      registeredUser.deviceToken.push(req.body.deviceToken);
+      registeredUser.save();
```

Comment:
There are
```
return User(registrationData).save();
```
in line 110.
If it's just adding deviceToken to array,
it would be good to combine with this.